Index.html

```diff
@@ -0,0 +1,189 @@
+            <h1>Your website is an investment in the success of your business.</h1>
+            <p>We are here to help you succeed.</p>
+            <button type="button">Let's work together</button>
+        </div>
+    <span class="image-header">
+    <img id="image header" src="assets/squiggle_line_stripe.svg" alt="squiggle line stripe"/>
+    </span>
+</header>
+
+<main class="outer-container">
+    <section class="outer-container">
+        <div class="inner-container container-section-1">
+            <p>Rekupe is a digital agency in Los Angeles.</p>
+            <p>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow. </p>
+            <p>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</p>
+        </div>
+        <div class="question">
+            <p>Have a question?<a href="click-here">Click here</a> to contact us!</p>
+        </div>
+    </section>
+
+    <section class="section-2 outer-container">
+        <div class="article-section-2">
+            <article class="article-settings-section-2">
+                <div class="article-settings-section-2-div">
+                    <h2>Rolling Stone</h2>
+                    <img src="assets/squiggle_line.svg" alt="squiggle line">
+                    <p>You know Rolling Stone. It’s the definitive
+                        voice in music, politics and culture. It’s a
+                        global icon.</p>
+                    <a href="https://viewproject.com">VIEW PROJECT</a>
+                </div>
+                <span class="article-image-section-2">
+                    <img src="assets/work-image-01.jpeg" alt="Rolling stone"/>
+                </span>
+
+            </article>
+            <article class="article-settings-section-2">
+                <span class="article-image-section-2">
+                    <img src="assets/work-image-02.jpeg" alt="Variety">
+                </span>
+                <div class="article-settings-section-2-div">
+                    <h3>Variety</h3>
+                    <img src="assets/squiggle_line.svg" alt="squiggle line"/>
+                    <p>Variety is an industry leader for breaking
+                        entertainment news, reviews and box
+                        office results with archive of content
+                        dating back to 1905.</p>
+                    <a href="https://viewproject.com">VIEW PROJECT</a>
+                </div>
+
+            </article>
+            <article class="article-settings-section-2">
+                <div class="article-settings-section-2-div">
+                    <h4>Worth</h4>
+                    <img src="assets/squiggle_line.svg" alt="squiggle line"/>
+                    <p>A global media platform connecting a
+                        community that embraces worth beyond
+                        wealth.</p>
+                    <a href="https://viewproject.com">VIEW PROJECT</a>
```

Comment:
Hoofdletters schrijven we nooit letterlijk in hoofdletters, maar transformeren we naar capitals in de CSS. Dat is toekomstbestendiger :)